ui/src/services/router/TuringRouter.js

```diff
@@ -109,19 +109,24 @@ export class TuringRouter {
     }
 
     // Ensembler
-    if (!!obj.config.ensembler) {
-      if (obj.config.ensembler.type === "nop") {
-        // Copy the final response route id to the top level, as the default route
-        obj.config.default_route_id =
-          obj.config["ensembler"].nop_config["final_response_route_id"];
-        delete obj.config["ensembler"];
-      } else if (obj.config.ensembler.type === "standard") {
-        // Copy the fallback response route id to the top level, as the default route
-        obj.config.default_route_id =
-          obj.config["ensembler"].standard_config["fallback_response_route_id"];
-        delete obj.config["ensembler"].standard_config[
-          "fallback_response_route_id"
-        ];
+    if (obj.config.ensembler.type === "nop") {
+      // Copy the final response route id to the top level, as the default route
+      obj.config.default_route_id =
+        obj.config["ensembler"].nop_config["final_response_route_id"];
+      delete obj.config["ensembler"];
+    } else if (obj.config.ensembler.type === "standard") {
+      // Copy the fallback response route id to the top level, as the default route
+      obj.config.default_route_id =
+        obj.config["ensembler"].standard_config["fallback_response_route_id"];
+      delete obj.config["ensembler"].standard_config[
+        "fallback_response_route_id"
+      ];
+    } else {
+      // Docker or Pyfunc ensembler, clear the default_route_id
```

Comment:
Would there be a need to clear `pyfunc_config` if the Ensembler type is Docker?

Reply:
Nope. Pyfunc ensemblers may also have docker configs (if it has been built), not the other way.